### PR TITLE
select status filter has options by default

### PR DESCRIPTION
**Pull request by sumith1896**
@martiansideofthemoon r+?

[Review on Reviewable](https://reviewable.io/reviews/jgraham/wptview/70)

### CODE REVIEW

**Comment by Manishearth** [comment]
-1 not enough jquery

File: angular_scripts.js [hunk 1]
```diff
@@ -242,9 +242,9 @@ app.controller('wptviewController', function($scope, ResultsModel) {
 
   $scope.addConstraint = function() {
     $scope.filter.statusFilter.push({
-      run : "",
+      run : $scope.runs[0].name,
       equality : "is",
-      status : [""]
+      status : ["pass"]
```
Comment (martiansideofthemoon) [inline angular_scripts.js]: Please refer the issue in this PR and change this to "PASS"

**Comment by martiansideofthemoon** [comment]
Looks good to me. **r+**. 
Thanks for your contribution :smile:

**Comment by sumith1896** [comment]
Thanks @martiansideofthemoon

**Comment by martiansideofthemoon** [comment]
fixes #69

**Pull request merged**